packages/sprite-tiling/src/TilingSpriteRenderer.ts

```diff
@@ -69,16 +69,15 @@ export class TilingSpriteRenderer extends ObjectRenderer
         vertices[2] = vertices[4] = (ts._width) * (1.0 - ts.anchor.x);
         vertices[5] = vertices[7] = ts._height * (1.0 - ts.anchor.y);
 
-        if (ts.uvRespectAnchor)
-        {
-            vertices = quad.uvs;
+        const { x: anchorX = 0, y: anchorY = 0 } = ts.uvRespectAnchor ? ts.anchor : {};
```

Comment:
This uses a lot of language features in one line! You _may_ want to consider not creating an object since this renderer runs on each ticker (especially in eager render loops):

```suggestion
        const anchorX = ts.uvRespectAnchor ? ts.anchor.x : 0;
        const anchorY = ts.uvRespectAnchor ? ts.anchor.y : 0;
```